CWE/decisions/Romania.txt

```diff
@@ -29,4 +30,195 @@ political_decisions = {
+			665 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 		
+			}
+			set_country_flag = romanian_oil_modernization_complete
+			treasury = -75000
+		}
+		ai_will_do = {
+			factor = 1
+		}
+	}
+
+
+
+	oil_modernization_hungarian_region = {
+		picture = "rom_oil"
+		potential = {
+			AND = {
+				owns = 2533
+				owns = 647
+			}
+			OR = {
+				has_country_flag = nicolae_ceausescu_flag
+				has_country_flag = rom_manescu
+				has_country_flag = gheorghe_apostol_romania
+				has_country_flag = rom_nicu
+				has_country_flag = rom_zoia
+			}
+			OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 government = socialist_democracy } }
+			has_country_flag = romanian_oil_modernization_start
+			NOT = { has_country_flag = romanian_oil_modernization_complete_hungarian_region }
+		}
+		
+		allow = {
+			NOT = { has_country_modifier = economic_malaise }
+			war = no
+			money = 25000
+		}
+		
+		effect = {
+			prestige = 50
+			2533 = {
+				trade_goods = natural_gas 
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 
+			}
+			647 = {
+				trade_goods = other_minerals
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+			}
+			set_country_flag = romanian_oil_modernization_complete_hungarian_region
+			treasury = -10000
+		}
+		ai_will_do = {
+			factor = 1
+		}
+	}
+
+
+	project_bucharest = {
+		picture = "renovate_capital"
+		potential = {
+			owns = 664
+			capital_scope = { has_province_modifier = new_capital }
+			NOT = { has_country_flag = germania }
+			NOT = { has_country_modifier = germania }
+			tag = ROM
+			OR = {
+				has_country_flag = nicolae_ceausescu_flag
+				has_country_flag = rom_manescu
+				has_country_flag = gheorghe_apostol_romania
+				has_country_flag = rom_nicu
+				has_country_flag = rom_zoia
+			}
+			OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 government = socialist_democracy } }
+			has_country_flag = systematization_ROM
+			NOT = { has_country_flag = germania }
+		}
+
+		allow = {
+			NOT = { has_country_modifier = economic_malaise }
+			war = no
+			money = 5000000
+			is_disarmed = no	
+			capital_scope = { has_building = railroad }
+		}
+
+		effect = {
+			prestige = 100
+			money = -2000000
+			money = -2000000
+			random_owned = { 
+				limit = { is_capital = yes } 
+				add_province_modifier = { name = peoples_palace duration = -1 } 
+			}
+			add_country_modifier = { name = germania duration = 3650 }
+			set_country_flag = germania
+			any_owned = { limit = { is_capital = no has_province_modifier = new_capital } life_rating = 5 remove_province_modifier = new_capital }
+		}
+		
+		ai_will_do = {
+			factor = 1
+		}
+	}
+
+
+	rom_decom_namechange = {
+		picture = "transition_to_democracy"
```

Comment:
Should have a picture of its own

Reply:
Do you mind if I have the decommunization decisions all use the same picture? I think Elia is making some for various post-Soviet states as well - so there will likely be a couple.

Reply:
I’m afraid I have to insist on different pictures for all decisions. This is to make it easier for players to identify decisions from the list. If the decision image is shared then it can be quite annoying for the player to find the correct decision out of the ones that look similar.

Reply:
I can't find unique pictures for every single nation, and can't look too deeply at the moment, so I'm just removing them for now.

Reply:
Okay